edge-proxy/src/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+# read service account token from filesystem
+file="/var/run/secrets/kubernetes.io/serviceaccount/token"
+if [[ -e ${file} ]]
+then
+    token=`cat ${file}`
+else
+    echo "Could not find file ${file}"
+fi
```

Comment:
Exit after this?

Reply:
Yes. It means that somebody tries to run this container, not in k8s environment. But I'm open to any suggestions